automation/repeated_test.sh

```diff
@@ -225,10 +240,12 @@ if [[ "${rwofs_sc}" == "local" ]]; then
 fi
 
 label_filter="(flake-check)||(${label_filter})"
-ginko_params="$ginko_params -no-color -succinct --label-filter="${label_filter}" -randomize-all"
-for test_file in $(echo "${NEW_TESTS}" | tr '|' '\n'); do
-    ginko_params+=" -focus-file=${test_file}"
-done
+ginko_params="$ginko_params -no-color -succinct --label-filter=${label_filter} -randomize-all"
```

Comment:
Maybe a good time to fix this typo? :) (`s/ginko_params/ginkgo_params/`)

Reply:
yeah, about time